package.json

```diff
@@ -42,6 +42,7 @@
     "typescript": "~1.6.2",
     "webpack": "~1.12.2",
     "webpack-config": "~1.0.2",
-    "webpack-dev-server": "~1.12.0"
+    "webpack-dev-server": "~1.12.0",
+    "webpack-stream": "^2.1.1"
```

Comment:
Thanks on your PR. You have one thing missing for changing gulp-webpack to webpack-stream. In glupfile.js you need also update ```var webpack = require('gulp-webpack');``` to ```var webpack = require('webpack-stream');```.